src/main/webapp/app/shared/error/error-boundary.tsx

```diff
@@ -31,6 +34,9 @@ class ErrorBoundary extends React.Component<IErrorBoundaryProps, IErrorBoundaryS
       ) : undefined;
       return (
         <div>
+          <Button tag={Link} className={'mb-2'} to="/" replace color="primary" onClick={() => this.setState(this.defaultState)}>
```

Comment:
What does the `replace` prop do?

Reply:
@calvinlu3 good question. I copied over from other pages. I looked into the component and can't find the prop. I will  remove in a separate PR.